js/src/lib/components/InferenceWidget/shared/helpers.ts

```diff
@@ -109,6 +110,7 @@ export async function getResponse<T>(
 	includeCredentials = false,
 	isOnLoadCall = false, // If true, the server will try to answer from cache and not do anything if not
 	useCache = true,
+	appendRepoPath = true,
```

Comment:
we should refactor the whole thing into a `params` object, this is really too much now

Reply:
I agree. In another branch ?